code/modules/clothing/spacesuits/rig/rig.dm

```diff
@@ -183,6 +186,10 @@
 /obj/item/weapon/rig/proc/reset()
 	offline = 2
 	flags &= ~NODROP
+	if(helmet&&helmet.on)
```

Comment:
This should have spacing between the operator and the variables. `if(helmet && helmet.on)`.